<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMICalc</class>
 <widget class="QDialog" name="BMICalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bmi Calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>weight:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="weightLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckBMIButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Check BMI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ResultEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="userBMIInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Your BMI:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>BMICalc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
